### PR TITLE
Add CI build on travisci

### PR DESCRIPTION
**Pull request by kimptoc**
See here for how to enable it for the quickfix-j user - http://docs.travis-ci.com/user/getting-started/

Please wait and see if this build of mine works first :)

https://travis-ci.org/kimptoc/quickfixj/builds/63240864

### CODE REVIEW

**Comment by kimptoc** [comment]
Hmm - it failed, will investigate tomorrow :)

**Comment by chrjohn** [comment]
Hmm, we also have Bamboo build here: http://www.quickfixj.org:8085/browse/QFJ-GIT 
:)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: java
+jdk:
+#  - oraclejdk8
+#  - oraclejdk7
+#  - openjdk7
+  - openjdk6
```
Comment (chrjohn) [inline .travis.yml]: Is there a reason why you chose openjdk instead of the Oracle one?

**Comment by guidomedina** [comment]
Try Oracle JDK 7 and remove/comment fork=true on the pom files since the build won't work with that option.

**Comment by kimptoc** [comment]
Whoops - did not notice the Bamboo build - Looks like there is a Bamboo plugin so that its status can be included around the web - https://marketplace.atlassian.com/plugins/com.atlassian.bamboo.plugins.atlassian-bamboo-plugin-build-status-image

**Comment by chrjohn** [comment]
@kimptoc Unfortunately, that plugin works only with Bamboo up to 5.2 but we have 5.4.

**Comment by kimptoc** [comment]
Don't know if you have a server to run this on (e.g. free on heroku :) ) - but this node app seems to do the job - https://github.com/kimptoc/bamboo-status

eg http://overzold1.kimptoc.net:8085/status/QFJ-GIT

**Comment by chrjohn** [comment]
Closed in favour of #121 
Sorry, @kimptoc , totally forgot about this one. :)

**Pull request closed without merging**